nix/distribution.nix

```
@@ -1,21 +1,24 @@
 {
   inputs,
   self,
-  version,
   ...
 }: {
   perSystem = {pkgs, ...}: {
-    packages = {
-      inherit (pkgs) prismlauncher-qt5-unwrapped prismlauncher-qt5 prismlauncher-unwrapped prismlauncher;
+    packages = let
+      ourPackages = self.overlays.default pkgs pkgs;
```

this is a little ugly and not very intuitive imo. `pkgs.extend` would probably be a better option

```suggestion
      ourPackages = pkgs.extend self.overlays.default;
```

---

This will cause nixpkgs to re-evaluate though

---

ah missed that part in the docs :p